src/cli/README_NETDATA.md

```diff
@@ -170,6 +172,39 @@ unpublish
 Done
 ```
 
+### full
+
+Usage: `netdata full`
+
+Print "yes" or "no" flag tracking whether or not the "net data full" callback has been invoked since start of Thread operation or since the last time `netdata full reset` was used to reset the flag.
+
+This command requires `OPENTHREAD_CONFIG_BORDER_ROUTER_SIGNAL_NETWORK_DATA_FULL`.
+
+The "net data full callback" is invoked whenever:
+
+- The device is acting as a leader and receives a Network Data registration from a Border Router (BR) that it cannot add to Network Data (running out of space).
+- The device is acting as a BR and new entries cannot be added to its local Network Data.
+- The device is acting as a BR and tries to register its local Network Data entries with the leader, but determines that its local entries will not fit.
+
+```
+> netdata full
+no
+Done
+```
+
+### full reset
+
+Usage: `netdata full reset`
+
+Reset the flag tracking whether "network data full" callback was invoked.
```

Comment:
"net data full" 

to be consistent with the other use above

Reply:
Good catch. Thanks. Updated in new push.